Graphs/TopoSortRecursive.js

```diff
@@ -0,0 +1,59 @@
+function TopoSortRecursiveFunction(graph, visited, stack, path, vertex) {
```

Comment:
Consider defining this function (as a closure) *inside* `TopoSortRecursive`. That way you don't need to pass `graph`, `visited`, `stack` and `path` around.

I'd also call this function `postorder`, because that's what it computes: Each node comes after its descendants (if there are no cycles).